test/couch_mrview_all_docs_tests.erl

```diff
@@ -107,6 +107,21 @@ should_query_with_limit_and_skip(Db) ->
     ]},
     ?_assertEqual(Expect, Result).
 
+should_query_with_config_limit_and_skip(Db) ->
```

Comment:
@tonysun83 Wait. We just figured out that `default_query_limit` doesn't reflects the behaviour.